rocketpy/rocket/aero_surface.py

```diff
@@ -349,12 +349,20 @@ def evaluate_geometrical_parameters(self):
         # If base radius is not given, the ratio between base radius and
         # rocket radius is assumed as 1, meaning that the nose cone has the
         # same radius as the rocket
-        if self.base_radius is None or self.rocket_radius is None:
+        if self.base_radius is None and self.rocket_radius is not None:
             self.radius_ratio = 1
+            self.base_radius = self.rocket_radius
+        elif self.base_radius is not None and self.rocket_radius is None:
+            self.radius_ratio = 1
+            self.rocket_radius = self.base_radius
         # If base radius is given, the ratio between base radius and rocket
         # radius is calculated
-        else:
+        elif self.base_radius is not None or self.rocket_radius is not None:
```

Comment:
You can only make the division if both `self.base_radius` **and** `self.rocket_radius` are not None.

```suggestion
        elif self.base_radius is not None and self.rocket_radius is not None:
```